haproxy/alb.tf

```
@@ -0,0 +1,51 @@
+resource "aws_lb" "haproxy_alb" {
+  name            = "haproxy-test-alb"
+  internal        = false
+  subnets         = ["${aws_subnet.haproxy_subnet.*.id}"]
+  security_groups = ["${aws_security_group.alb.id}"]
+
+  tags {
+    Name = "haproxy_alb"
+  }
+}
+
+resource "aws_lb_target_group" "haproxy_alb_target" {
+  name     = "haproxy-test-alb-tg"
+  vpc_id   = "${aws_vpc.default.id}"
+  protocol = "HTTP"
+  port     = 80
+
```

Ran `terraform plan` and it resulted in the following:
```
Terraform will perform the following actions:

  ~ aws_lb_target_group.haproxy_alb_target
      proxy_protocol_v2: "" => "false"


Plan: 0 to add, 1 to change, 0 to destroy.
```
Perhaps:
```suggestion
  proxy_protocol_v2 = "false"
```